we have a ton of broken dependency chains in aports;
let's ignore them, so the build can succeed with a dirty world

--- a/python/mach/mach/site.py
+++ b/python/mach/mach/site.py
@@ -1155,6 +1155,7 @@
         'Running "pip check" to verify compatibility between the system Python and the '
         f'"{virtualenv_name}" site.'
     )
+    return
 
     with tempfile.TemporaryDirectory() as check_env_path:
         # Pip detects packages on the "sys.path" that have a ".dist-info" or
